docs/3.0.0/docs/canton/usermanual/pruning.rst

```diff
@@ -43,6 +43,69 @@ For the maximum duration to specify a reliable pruning window "end time", the le
 should should not be wildcards (`*`) as illustrated in the examples above. If the hour field is fixed, so should the
 fields for minute and second.
 
+Schedule Format
+---------------
+
+The format used to describe cron expression consists of 7 whitespace-separated fields:
+
+#. seconds
+#. minutes
+#. hours
+#. day-of-month
+#. month
+#. day-of-week
+#. year (optional)
+
+Each field has a set of valid values:
+
+* seconds and minutes must be a number between ``0`` and ``59``
+
+* hours must be a number between ``0`` and ``23``
+
+* day of the month must be a number between ``1`` and ``31``
+
+* month must be either
+
+  #. a number between ``0`` and ``11`` (with ``0`` being January), or
+
+  #. the first three letters of the month name (``JAN``, ``FEB``, ``MAR``, ...)
+
+* day of the week must be either
+
+  #. a number between ``1`` and ``7`` (with ``1`` being Sunday), or
+
+  #. the first three letters of the weekday name (``SUN``, ``MON``, ``TUE``, ...)
+
+* year must be a number between ``1900`` and ``2099``
+
+Note that while a day-of-month value might be valid by the definition above, it might not exist at all (like the
+31st of November). If you set a schedule to tick every month on the 31st, every month with less than 31 days will
+be skipped.
+
+You can construct lists and ranges of values. For example, the day of the week could be a range like ``MON-FRI``
+to refer to the days Monday through Friday, or ``TUE,FRI`` to refer to Tuesday or Friday exclusively, or a mix of
+both, for example ``MON,WED-FRI``, meaning "Monday and Wednesday through Friday".
+
+The ``*`` character is called a wildcard and it means "all possible values" (compare it with the example above, where
+the wildcard character in the month position means "every month").
+
+The ``?`` character can be used in either the day-of-month and day-of-week fields, so that either field can be used
+freely. For example, if you want to specify "every Monday at noon", the ``?`` character can be used to mean that you
+don't want to restrict the day-of-month as follows: ``0 0 12 ? * MON``.
+
+The ``/`` character can be applied to numeric values to specify increments. For example, ``1/2`` in the hours
+field it means "every two hours starting from 1 AM", i.e 1 AM, 3 AM, 5 AM, etc..
```

Comment:
```suggestion

Advanced schedule formatting
****************************

* You can construct **lists and ranges** of values. For example, the day of the week could be a range like ``MON-FRI`` to refer to the days Monday through Friday, or ``TUE,FRI`` to refer to Tuesday and Friday exclusively. Or you could use a mix of both, for example, ``MON,WED-FRI``, meaning "Monday, and also Wednesday through Friday."
* Use the **asterisk** (``*``) as a wildcard that means "all possible values."
* Use the **question mark** (``?``) as a wildcard that means "any value" in the day-of-the-month and day-of-the-week fields. For example, to specify "every Monday at noon," use the ``?`` character to indicate that any day of the month is valid: ``0 0 12 ? * MON``
* To apply **increments** to numeric values, use the slash character (``/``). For example, a value of ``1/2`` in the hours field means "every two hours starting from 1 AM" (1 AM, 3 AM, 5 AM, ...).

```

It should look like this:

<img width="777" alt="Screenshot 2023-12-03 at 7 08 46 PM" src="https://github.com/digital-asset/docs.daml.com/assets/139921049/12517f0c-1e6b-42cc-8173-456d53c1707e">